app/services/food_service.rb

```diff
@@ -0,0 +1,10 @@
+class FoodService
+  def self.find_foods_by_ingredient(food)
+    response = Faraday.get("https://api.nal.usda.gov/fdc/v1/foods/search?") do |faraday|
+      faraday.params[:query] = "#{food}"
+      faraday.params[:pageSize] = 10
+      faraday.params[:api_key] = "#{ENV['api_key']}"
+    end
+    parse = JSON.parse(response.body, symbolize_names: true)
+  end
+end
```

Comment:
This all looks good to me -- I know we talk quite a bit about abstracting the connection, but in the case of this mid mod, when we've asked for nothing other than this one user story, there wasn't really a need for that. As soon as you needed to make another call to that api, that's when I'd make sure to abstract the connection into it's own method. I also like how you're including the pageSize in the query -- nice job reading the docs.